mirror_list.txt

```diff
@@ -1,5 +1,3 @@
-golang:latest
-node:lts-slim
-node:lts
-composer:2.1.11
-python:slim
```

Comment:
`python:slim` still mentioned [in the python stack's Dockerfile](https://github.com/devfile/registry/blob/main/stacks/python/3.0.0/docker/Dockerfile#L1)